pandas/core/algorithms.py

```diff
@@ -1105,6 +1105,22 @@ def _get_score(at):
         return result
 
 
+def check_percentile(q):
```

Comment:
Can you annotate this function? I think `q` should just be `float, Iterable[float]`

Reply:
Return should be an ndarray

Reply:
Done.